src/translate.js

```diff
@@ -84,20 +84,35 @@ function uploadRemoteTranslations(auth, languages, translations) {
     rows.push([k, ...languages.map((l) => translation[l])]);
   });
 
-  log('Uploading new translations');
+  log("Uploading new translations");
   return uploadSheetData(auth, rows);
 }
 
 function exportTranslations(translations) {
-  fs.writeFileSync(
-    config.translationsPath,
-    JSON.stringify(translations, null, 2),
-  );
+  if (config.locales) {
+    config.locales.forEach((locale) => {
```

Comment:
maybe adding an option to cleanup the flatten file?

Reply:
How about cleaning the flatten file after exported the translation files? @romitamgai